TACC/tup-ui
# feat: allow smaller `<TicketCreateModal>` button

## Overview / Changes

1. Allow `TicketCreateModal` to use different size (or type) buttons.
2. Reduce size of [ + New Ticket ] on Dashboard.

## Related

- required by #317
- supersedes #318
- simplified by #320

## Testing

1. Open Dashboard.
2. Verify "+ New Ticket" button is smaller.
3. Thus, see height of header title and button height are less different.

## UI

| before | after |
| - | - |
| ![before](https://github.com/TACC/tup-ui/assets/62723358/de36ff4e-4ff8-4d0f-af05-ade1b7414915) | ![after](https://github.com/TACC/tup-ui/assets/62723358/fb2a0d31-2c81-4bb8-a470-96ea30f11a53) |
